converters/json/src/test/java/io/smallrye/config/converter/json/JsonArrayConverterTest.java

```diff
@@ -33,12 +33,14 @@
 
     @Deployment
     public static WebArchive createDeployment() {
-        final File[] thorntailMPConfigFiles = Maven.resolver().resolve("io.thorntail:microprofile-config:2.4.0.Final")
```

Comment:
Sorry @phillip-kruger, can we have the fix of removing Thorntail and switching to SmallRye Config in a separate PR, as it's unrelated to adding Config Events. Thanks

Reply:
Sorry, yes will do a separate PR. Reverted.